ci/values.yml

```diff
@@ -3,6 +3,7 @@
 deployments_git_uri: git@github.com:GaloyMoney/galoy-deployments.git
 git_uri: git@github.com:GaloyMoney/galoy-infra.git
 git_branch: main
+git_version_branch: testflight-uid
```

Comment:
I would make this more specific. Its not the global version - its specifically the `testflight_uid_branch` or something like that.